src/pageLayouts/ilite/IliteHome/iliteHome.import.less

```diff
@@ -152,6 +152,10 @@ html {
         flex-direction: column;
       }
     }
+
+    .iliteHome__news-button {
+      margin-top: 2.5%;
```

Comment:
@Spartan104 Top & bottom margins should be set to a static value, like `px`. We want left & right to grow with the page width, but the top and bottom should remain static since the height scrolls instead of stretches.